### ISSUE TITLE
Tensorboard -summaries is not capturing train and validation loss,WER 

### ISSUE THREAD

**Issue by pdcoded** [mozilla/DeepSpeech#517]
I am running tensorboard after a batch of training . But I dont see the loss curve over epochs. Do I have to specifically add that summary after each epoch run ?

**Comment by kdavis-mozilla** [comment]
Currently there's no variable for `loss`. So `log_variable()` does not log the value.

If you want `loss` to appear in Tensorboard , you have to explicitly add a `loss` summary. A similar statement is true of WER.

**Comment by Franck-Dernoncourt** [comment]
I agree that it'd be useful to log loss and WER. 

https://github.com/utunga/DeepSpeech/pull/1/commits/88fff498a036e03ce492bf6bfd340749a421c2df is a commit adding loss logging.

**Comment by reuben** [comment]
Fixed by #1391.

**Comment by ambigus9** [comment]
@reuben  Hi, any way to see **log_step_loss** or **WER** on Tensorboard?

Currently I'm trying to view the values of checkpoints on Tensorboard while is training: `tensorboard --logdir=checkpoints/output/`

**Comment by reuben** [comment]
I don't know what log_step_loss is but we do log train and validation step losses in Tensorboard. Make sure you're pointing it to the same folder you passed to DeepSpeech.py as `--summary_dir`.

**Comment by ambigus9** [comment]
Thanks @reuben So, it should be: `tensorboard --logdir=/home/miguel/voice-to-text/checkpoints/output32/ --summary_dir=/home/miguel/DeepSpeech-lfs/DeepSpeech/` ?

**Comment by reuben** [comment]
No, I meant Tensorboard's logdir should be the same as DeepSpeech's summary_dir.

**Comment by ambigus9** [comment]
@reuben Sure, here are some variables that i see on tensorboard: 
![image](https://user-images.githubusercontent.com/16579856/50528449-56543780-0abc-11e9-8bf9-4a967f446b40.png)
Should i see this variable?

![image](https://user-images.githubusercontent.com/16579856/50528463-69670780-0abc-11e9-9900-6fb2a0b91f93.png)

**Comment by reuben** [comment]
Looks like you're running a very old version of the code (you should have lead with that), which doesn't log the individual step losses.

**Comment by reuben** [comment]
This has nothing to do with the client version. The client is for inference, but you're doing training, so you should upgrade your DeepSpeech *code* to a more recent version.

**Comment by ambigus9** [comment]
Thanks! I will update my DeepSpeech code!
